core/schema_utils.py

```diff
@@ -67,9 +68,22 @@
 SCHEMA_TYPE_BASESTRING = 'basestring'
 SCHEMA_TYPE_UNICODE_OR_NONE = 'unicode_or_none'
 SCHEMA_TYPE_OBJECT_DICT = 'object_dict'
+SCHEMA_TYPE_WEAK_MULTIPLE = 'weak_multiple'
 
 SCHEMA_OBJ_TYPE_SUBTITLED_HTML = 'SubtitledHtml'
 SCHEMA_OBJ_TYPE_SUBTITLED_UNICODE = 'SubtitledUnicode'
+ALL_SCHEMAS: Dict[str, Union[type, Tuple[Union[type, Tuple[Any, ...]], ...]]] = { # pylint: disable=line-too-long
```

Comment:
Why is the type so complicated? Cannot it be simplified?

Reply:
Yep simplified it